metis-enrichment/metis-enrichment-service/src/main/java/eu/europeana/enrichment/service/zoho/ZohoAccessService.java

```diff
@@ -159,19 +161,24 @@ Organization getOrganizationFromJsonNode(JsonNode jsonRecord)
 	String toEdmCountry(String organizationCountry) {
 		if (StringUtils.isBlank(organizationCountry))
 			return null;
-		int separatorPos = organizationCountry.indexOf("(");
-		if (separatorPos < 0)
-			return organizationCountry;
+
+		int commaSeparatorPos = organizationCountry.indexOf(",");
+		int bracketSeparatorPos = organizationCountry.indexOf("(");
+
+		if (commaSeparatorPos > 0)
```

Comment:
Can you create a local variable where the if/else statements will set it and then return that variable? This way the return statements will be much less.

Reply:
Not really .. the problem is that the Zoho gets updated by changing the country from FR(France) to France, FR format. I can write a TODO to remove when the data gets consistent again.